### DISCUSSION TITLE
What are the differences between IBM Plex Mono and Zed Plex Mono?

### DISCUSSION THREAD

**Discussion by gh-nate** [zed-industries/zed#14405] [Feature Requests]
Zed Plex Mono is referenced in https://zed.dev/docs/configuring-zed#buffer-font-family

Whilst in https://zed.dev/releases/stable/0.143.6

> Changed the default fonts to IBM Plex Sans and IBM Plex Mono.

**Comment by gh-nate** [comment]
I did some more digging and found https://github.com/zed-industries/zed/commit/7d7fd7d25d6efe5898cb9f92e46828ffad4f001e which still doesn't really address the need for difference in naming.

---

The license states:

> 3) No Modified Version of the Font Software may use the Reserved Font Name(s) unless explicit written permission is granted by the corresponding Copyright Holder. This restriction only applies to the primary font name as presented to the users.

If there were modifications, they don't seem to be explicitly stated anywhere...

---

I can't be bothered to compare checksums of the Zed Plex Mono files across the IBM Plex Mono versions to determine whether there were modifications.

**Comment by rofrol** [comment]
introduced here https://github.com/zed-industries/zed/pull/13596

**Comment by josh-nz** [comment]
Zed Plex Mono adds ligatures apparently. IBM Plex Mono does not have ligatures. Refer https://github.com/idr4n/github-monochrome.nvim/issues/1#issuecomment-2620737434

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
